openpower/package/openpower-pnor/Config.in

```diff
@@ -112,3 +114,11 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
 config BR2_OPENPOWER_PNOR_XZ_ENABLED
         bool "Enable xz compression in PNOR payloads"
         default n
+
+config BR2_OPENBMC_SYSTEM
+        bool "Build for systems using OpenBMC"
+        default n
+
+config BR2_BUILD_VPNOR
```

Comment:
Is there a name for the file format rather than what consumes it?

e.g. if we were adding support for constructing a HPM image, we'd have "BR2_BUILD_HPM" or something